README.md

```diff
@@ -168,3 +168,5 @@ Ports used during development:
 * 53333 - Stencil
 * 56106 - Storybook for Web Components
 * 56206 - Storybook for CSS components
+* 56406 - Storybook for React components
```

Comment:
Moet er niet ook nog iets van tekst en uitleg worden toegevoegd aan de sectie `development` in deze readme?

Reply:
Ja klopt